src/backend/joanie/core/models/products.py

```diff
@@ -362,9 +362,30 @@ def delete(self, using=None, keep_parents=False):
         webhooks.synchronize_course_runs(serialized_course_runs)
 
 
+class OrderGroupManager(models.Manager):
+    """Custom manager for the OrderGroup model."""
+
+    def find_assignables(self, course_product_relation_id):
```

Comment:
IMO we should not use the `assignable` term here as it could be confusing with the method `OrderGroup.is_assignable`. Indeed `find_assignable` is looking for active order group for the given relation while `is_assignable` is checking if the order group is enabled and has still available seats.

What do you think to rename it `find_actives` ?